behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/uttak/fp/UttakStegBeregnStønadskontoTjeneste.java

```diff
@@ -68,8 +67,7 @@ public class UttakStegBeregnStønadskontoTjeneste {
         // Endring fra 80 til 100% DG krever full omregning ettersom antall dager reduseres
         var endretDekningsgrad = dekningsgradTjeneste.behandlingHarEndretDekningsgrad(ref);
         var fullBeregning = endretDekningsgrad && Dekningsgrad._100.equals(dekningsgradTjeneste.finnGjeldendeDekningsgrad(ref));
-        var eksisterendeKontoUtregning = fagsakRelasjon.getGjeldendeStønadskontoberegning().orElseThrow();
-        if (endretDekningsgrad || skalBeregneMedPrematurdager(fpGrunnlag, eksisterendeKontoUtregning)) {
+        if (endretDekningsgrad) {
             beregnStønadskontoerTjeneste.overstyrStønadskontoberegning(input, !fullBeregning);
             return BeregningingAvStønadskontoResultat.OVERSTYRT;
         }
```

Comment:
public Stønadskontoberegning fastsettStønadskontoerForBehandling(UttakInput input) {
        var ref = input.getBehandlingReferanse();
        var gjeldendeStønadskontoberegning = fagsakRelasjonTjeneste.finnRelasjonFor(input.getBehandlingReferanse().saksnummer())
            .getGjeldendeStønadskontoberegning();
        return beregnStønadskontoerTjeneste.beregnForBehandling(input, gjeldendeStønadskontoberegning.orElseThrow().getStønadskontoutregning())
            .or(() -> gjeldendeStønadskontoberegning)
            .orElseThrow();
    }

Metoden nedenfor som heter fastsettStønadskontoerForBehandling() har litt logikk for å sjekke diff som virker litt unødvendig ut. Hvis resultatet er likt kan vi ikke bare returener ny beregning alltid? Enten er det likt eller ulikt.

Reply:
Kan deretter forenkle StønadskontoRegelAdapter ganske mye. (fjerne metodene 
```
private boolean endretUtregning(Map<StønadskontoType, Integer> tidligereUtregning, StønadskontoResultat resultat){}
public Optional<Stønadskontoberegning> beregnKontoerSjekkDiff(...) {}
```

Reply:
Vi kan ikke regne på nytt i alle behandlinger før vi har stabilisert input på tvers av sakskompleks med AP + OVST mm.
Problemsakene kjennetegnes ved at de to partene har oppgitt ulike data (eller ombestemmer seg).
Til vi får input stabilt så gjør vi utregning relativt til det "første sak" har lagt i fagsakrelasjon.

Reply:
Dessuten skal kontostrukturen være låst inntil den evt eksplisitt tredeles (til kvoter) - ikke endres pga søknad etter vedtak eller stebarnsadopsjon. 
Får utvikle dette mer når ting er på plass